docs/notebooks/Getting_started_1.ipynb

```diff
@@ -0,0 +1,440 @@
+    "\n",
+    "Below is a 1-Lipschitz replication of the previous MLP toy-example, using `deel-lip`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "a62d3a3f-c0e8-4a3e-9025-758afebf99bd",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import deel\n",
+    "from deel import lip"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "c26f52a6-c2ec-49b5-a99f-3353dc3f3044",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "C:\\Users\\kierszbaums\\anaconda.related\\envs\\1_lipschitz\\deel_lip\\lib\\site-packages\\keras\\src\\initializers\\initializers.py:120: UserWarning: The initializer Orthogonal is unseeded and being called multiple times, which will return identical values each time (even if the initializer is unseeded). Please update your code to provide a seed to the initializer, or avoid using the same initializer instance more than once.\n",
+      "  warnings.warn(\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Model: \"sequential_1\"\n",
+      "_________________________________________________________________\n",
+      " Layer (type)                Output Shape              Param #   \n",
+      "=================================================================\n",
+      " flatten_2 (Flatten)         (None, 784)               0         \n",
+      "                                                                 \n",
+      " spectral_dense (SpectralDe  (None, 64)                100481    \n",
+      " nse)                                                            \n",
+      "                                                                 \n",
+      " group_sort2 (GroupSort2)    (None, 64)                0         \n",
+      "                                                                 \n",
+      " spectral_dense_1 (Spectral  (None, 32)                4161      \n",
+      " Dense)                                                          \n",
+      "                                                                 \n",
+      " group_sort2_1 (GroupSort2)  (None, 32)                0         \n",
+      "                                                                 \n",
+      " spectral_dense_2 (Spectral  (None, 10)                661       \n",
+      " Dense)                                                          \n",
+      "                                                                 \n",
+      "=================================================================\n",
+      "Total params: 105303 (411.34 KB)\n",
+      "Trainable params: 52650 (205.66 KB)\n",
+      "Non-trainable params: 52653 (205.68 KB)\n",
+      "_________________________________________________________________\n"
+     ]
+    }
+   ],
+   "source": [
+    "L1_model = lip.model.Sequential([    \n",
```

Comment:
`L1_model` sounds to me like $L_1$ norm. Maybe a more explicit name, like `Lip_model`?